unitary/examples/quantum_chinese_chess/enums.py

```diff
@@ -42,31 +47,38 @@ class MoveVariant(enum.Enum):
     CAPTURE = 3
 
 
-class Piece(enum.Enum):
-    EMPTY = "."
-    SOLDIER = "s"
-    CANNON = "c"
-    ROOK = "r"
-    HORSE = "h"
-    ELEPHANT = "e"
-    ADVISOR = "a"
-    GENERAL = "g"
+class Color(enum.Enum):
+    NA = 0
+    RED = 1
+    BLACK = 2
 
-    @classmethod
-    def type_of(cls, c: str) -> Optional["Piece"]:
-        return {
-            "s": Piece.SOLDIER,
-            "c": Piece.CANNON,
-            "r": Piece.ROOK,
-            "h": Piece.HORSE,
-            "e": Piece.ELEPHANT,
-            "a": Piece.ADVISOR,
-            "g": Piece.GENERAL,
-            ".": Piece.EMPTY,
-        }.get(c.lower(), None)
 
-    def red_symbol(self) -> str:
-        return self.value.upper()
+class Type(enum.Enum):
+    """The four values are symbols corresponding to
+    English red
+    English black
+    Chinese red
+    Chinese black
+    """
 
-    def black_symbol(self) -> str:
-        return self.value
+    EMPTY = [".", ".", ".", "."]
```

Comment:
I would probably use tuples instead of lists here, since you do not want them to be mutable.

Reply:
Thanks Doug. It's updated.